public/app/plugins/datasource/grafana-azure-monitor-datasource/resourcePicker/resourcePickerData.ts

```diff
@@ -64,6 +69,45 @@ export default class ResourcePickerData extends DataSourceWithBackend<AzureMonit
     return addResources(rows, parentRow.uri, nestedRows);
   }
 
+  search = async (searchPhrase: string, searchType: 'logs' | 'metrics'): Promise<ResourceRowGroup> => {
+    const searchQuery = {
```

Comment:
I get why the query is different for metrics and logs but maybe it shouldn't? Checking the Azure portal again, it seems that the behavior there is to include resource groups (and subs) there. The case is different because there it allows you to further select a resource type and query multiple resources (which we don't support for metrics):

![Screenshot from 2022-04-28 12-48-36](https://user-images.githubusercontent.com/4025665/165736608-375dd35d-94f5-4071-ad8a-d3a6102d239f.png)

I would re-use the logs query because:
 - It is get us closer to the portal approach, if we add support to query multiple resources for metrics
 - If someone was looking for a resource-group (even if they cannot use it), it would be confusing to skip that value.

So for the current state, I would list the resource but make it non selectable. Does that make sense?